### ISSUE TITLE
Error deploying when update to 5f468bb437df

### ISSUE THREAD

**Issue by thib1984** [flathub/org.gnome.Evolution#39]
Since two weeks, when i run the command 
```flatpak update org.gnome.Evolution```
I obtain    
```
Looking for updates...
Updating in system:
org.gnome.Evolution/x86_64/stable flathub 5f468bb437df
Is this ok [y/n]: y
Updating: org.gnome.Evolution/x86_64/stable from flathub
[####################] 1 metadata, 0 content objects fetched; 592 B transferred in 2 seconds
Error: Failed to update org.gnome.Evolution/x86_64/stable: Error deploying: No such file or directory
error: There were one or more errors
```

**Comment by Sesivany** [comment]
Not sure if this is a bug in Evolution. It seems like something broken in your local flatpak repos. I would check flatpak/ostree logs.
We haven't received any other complaint about updates.

**Comment by mochiballs** [comment]
I get the same error when running 
`flatpak update`

**Comment by felipeborges** [comment]
Have you tried `flatpak repair`? https://docs.flatpak.org/en/latest/flatpak-command-reference.html#flatpak-repair

**Comment by thib1984** [comment]
Yes, and i tried too to uninstall app, but : 

```
~ flatpak install flathub org.gnome.Evolution 
Installing in system:
org.gnome.Evolution/x86_64/stable        flathub 5f468bb437df
  permissions: ipc, network, pulseaudio, wayland, x11, dri
  file access: xdg-run/dconf, ~/.config/dconf:ro, ~/.gnupg
  dbus access: ca.desrt.dconf, org.freedesktop.Notifications, org.freedesktop.secrets, org.gnome.OnlineAccounts, org.gnome.keyring.SystemPrompter
org.gnome.Evolution.Locale/x86_64/stable flathub 28a34e927f71
Is this ok [y/n]: y
Installing: org.gnome.Evolution/x86_64/stable from flathub
[####################] 253 metadata, 4796 content objects fetched; 16363 KiB transferred in 10 seconds
error: Failed to install org.gnome.Evolution/x86_64/stable: Error deploying: Aucun fichier ou dossier de ce type
~ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.5 LTS
Release:	18.04
Codename:	bionic
~ uname -a 
Linux PY9786 4.15.0-118-generic #119-Ubuntu SMP Tue Sep 8 12:30:01 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux

```

**Comment by mochiballs** [comment]
> Have you tried `flatpak repair`? https://docs.flatpak.org/en/latest/flatpak-command-reference.html#flatpak-repair

Hi Felipe, thanks for the suggestion. I had tried `flatpak repair`, `flatpak repair --system`, and `flatpak repair --user` before running `flatpak update` and still get the same error message

**Comment by Sesivany** [comment]
I discussed it with Alex, the lead developer of Flatpak and he himself has no idea where the problem could be. It just seems based on the output that it's on the client side.
While doing the update operation can you please run this command in another terminal? /usr/libexec/flatpak-system-helper -v --no-idle-exit --replace
And if you try to install the app again, can you try it with --user to see if it makes any difference?

**Comment by alexlarsson** [comment]
Not  just the flatpak-system-helper thing, but also pass `-vv` to the update command itself.

**Comment by mverouden** [comment]
@Sesivany @alexlarsson Same Issue on my system:
`Looking for updates...
Updating in system:
org.gnome.Evolution/x86_64/stable flathub 5f468bb437df
Is this ok [y/n]: y
Updating: org.gnome.Evolution/x86_64/stable from flathub
[####################] 1 metadata, 0 content objects fetched; 592 B transferred 
Error: Failed to update org.gnome.Evolution/x86_64/stable: Error deploying: No such file or directory
error: There were one or more errors`

Have tried all suggested commands and still get the same message. My systeminfo (cat etc/os-release):
`cat /etc/os-release
NAME="Linux Mint"
VERSION="19.3 (Tricia)"
ID=linuxmint
ID_LIKE=ubuntu
PRETTY_NAME="Linux Mint 19.3"
VERSION_ID="19.3"
HOME_URL="https://www.linuxmint.com/"
SUPPORT_URL="https://forums.linuxmint.com/"
BUG_REPORT_URL="http://linuxmint-troubleshooting-guide.readthedocs.io/en/latest/"
PRIVACY_POLICY_URL="https://www.linuxmint.com/"
VERSION_CODENAME=tricia
UBUNTU_CODENAME=bionic`

Suggestion to fix this are highly appreciated! Thanks in advance.

**Comment by mverouden** [comment]
Just ran: flatpak update -vv, the last part of the output might be interesting (but not helpful to me at least):
`Updating in system:
org.gnome.Evolution/x86_64/stable flathub 5f468bb437df
Is this ok [y/n]: y
Updating: org.gnome.Evolution/x86_64/stable from flathub
[####################] 1 metadata, 0 content objects fetched; 592 B transferred F: Calling system helper: Deploy

Error: Failed to update org.gnome.Evolution/x86_64/stable: Error deploying: No such file or directory
error: There were one or more errors`

**Comment by thib1984** [comment]
I see that at least two os us use ubuntu bionic or derivates. Since few days, i switch to ubuntu focal and this bug does not appear.

**Comment by mverouden** [comment]
@thi1984 That's exactly what I wanted to try next. I suppose it is an issue with the flatpak version installed in Bionic or derivatives of it (e.g. Linux Mint 19).

When I have time, I will try to reinstall my production desktop at University. I tried today to update Evolution there and it displayed the same issue. It also runs on Linux Mint 19.3. Deleted Evolution flatpak and tried to install the latest version, unfortunately Evolution version 3.38.0 as flatpak won't even install. Currently because of SARS-CoV-2 I am mainly using it remotely for calculations, therefore I am not needing Evolution on that machine at the moment.

@Sesivany @Sesivany It seems to be a problem with the flatpak version in Ubuntu 18.04 LTS (Bionic). It just will not deploy Evolution flatpak v3.38.0. Might be something to look into, because Bionic is still supported as LTS until 2023!

**Comment by thib1984** [comment]
@K4rnival  What is your distribution if you reproduce the bug?

**Comment by K4rnival** [comment]
@thib1984 Tried it on Ubuntu 18.04

**Comment by izahn** [comment]
I also have this problem on RHEL 7.5.

**Comment by mverouden** [comment]
Dear all,

I solved the issue on Linux Mint 19.3 (Ubuntu 18.04 LTS based).

Here is what I did:

- Upgrade flatpak to the latest edition (see flatpak.org)
- uninstall all flatpak applications
- run: flatpak uninstall --unused
- check that no appliations are left with: flatpak list

Now reinstall Evolution via: flatpak install flathub org.gnome.Evolution

Kind regards,
